### PR TITLE
two tiny tweaks to the docs

### PR DESCRIPTION
**Pull request by odigity**
- update first line of README.md to match GitHub project description
- update docs link in README.md to point to RTD.io
- add link to docs to setup.cfg so it shows up in PyPI

### CODE REVIEW

**Comment by odigity** [comment]
I hope this is helpful - and I hope it did it correctly.  (I haven't done much contributing to FOSS.)

File: setup.cfg [hunk 1]
```diff
@@ -9,6 +9,7 @@ long_description_content_type = text/markdown
 url = https://github.com/miguelgrinberg/flask-httpauth
 project_urls =
     Bug Tracker = https://github.com/miguelgrinberg/flask-httpauth/issues
+    Documentation = http://pythonhosted.org/Flask-HTTPAuth
```
Comment (miguelgrinberg) [inline setup.cfg:12]: The current documentation link is https://flask-httpauth.readthedocs.io/. Would you please update the README.md file with it? Thanks!

Comment (odigity) [reply setup.cfg]: Done!

**Comment by odigity** [comment]
BTW - I just discovered this package a couple hours ago, and am still reading through the docs.  I just noticed this bit:

```
@auth.verify_password
def verify_password(username, password):
    if username in users and \
            check_password_hash(users.get(username), password):
        return username

...

"...the function should return the user object."
```
(ref: https://flask-httpauth.readthedocs.io/en/latest/#basic-authentication-examples)

The docs say to return a User object, but the example demonstrates returning a username - and later in the same example, `current_user()` is called, which is supposed to return the User object that was returned from `verify_password`, but it's just a username...

Is this an error in the docs?  I can fix it in the same PR...

**Comment by miguelgrinberg** [comment]
> The docs say to return a User object

This package does not care how you represent your users. Whatever you return is what you are going to get when you call `current_user()`. A string is an acceptable "User object" for simple applications.

> The get_password method is marked deprecated, but is still used in the Digest Auth example.

The `get_password()` method is deprecated for Basic auth. It is not deprecated for Digest auth.

**Comment by miguelgrinberg** [comment]
Thanks!

**Pull request merged**